extensions/support-page/main.ts

```diff
@@ -7,7 +7,7 @@ export default class SupportPageMainExtension extends LensMainExtension {
       parentId: "help",
       label: "Support",
       click() {
-        windowManager.navigate(supportPageURL());
+        Navigation.navigate(supportPageURL());
```

Comment:
I'm wondering if it's possible to wrap this as a method in the `LensExtension` class so this line could be
```suggestion
        this.navigate(supportPageURL());
```
and no need to import `Navigation` (and maybe even don't need to expose it in the extensions api?)

Reply:
We could even have a `LensExtension` `navigate()` method that accepts a page id (which I guess is essentially the `PageRegistration` `path` field)

Reply:
Yes, eventually I had to do like you suggested .. otherwise prod build fails.